packages/react-button/src/components/MenuButton/MenuButton.types.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { ComposeOptions } from '@fluentui/react-compose';
+import { ComposeStandardStatics, ShorthandValue } from '../../utils/tempTypes';
+import { ButtonProps, ButtonTokens } from '../Button/Button.types';
+
+export interface MenuButtonProps extends Omit<ButtonProps, 'icon' | 'iconPosition' | 'loader'> {
+  /**
```

Comment:
Out of curiosity, why can't a menu button have an icon to the left?

Reply:
I wanted to omit extra icons when having a `MenuButton` besides the `menuIcon` itself.

This brings a good point of discussion though:
- Should we allow the `menuIcon` to be on the left as well?
- Should we allow for an `icon` besides the `menuIcon`?
- If so, should we allow that `icon` to be able to exist before and after the content or just before?